test/cli/install/bun-patch.test.ts

```diff
@@ -90,10 +90,21 @@ describe("bun patch <pkg>", async () => {
 
     describe("inside ROOT workspace package", async () => {
       const args = [
-        ["packages/eslint-config/node_modules/@types/ws", "packages/eslint-config/node_modules/@types/ws"],
-        ["@types/ws@8.5.4", "node_modules/@repo/eslint-config/node_modules/@types/ws"],
+        [
+          "packages/eslint-config/node_modules/@types/ws",
+          "packages/eslint-config/node_modules/@types/ws",
+          "@types/ws@8.5.4",
+          "patches/@types%2Fws@8.5.4.patch",
+        ],
+        [
+          "@types/ws@8.5.4",
+          "node_modules/@repo/eslint-config/node_modules/@types/ws",
+          "@types/ws@8.5.4",
+          "patches/@types%2Fws@8.5.4.patch",
+        ],
+        ["@types/ws@7.4.7", "node_modules/@types/ws", "@types/ws@7.4.7", "patches/@types%2Fws@7.4.7.patch"],
       ];
-      for (const [arg, path] of args) {
+      for (const [arg, path, ver, patch_path] of args) {
```

Comment:
The reader might not know what `ver` means, the shorthand isn't much shorter

```suggestion
      for (const [arg, path, version, patch_path] of args) {
```